release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2457,6 +2438,38 @@ spec:
 
 * The pipeline metrics API does not support the required `pipelinerun/taskrun` histogram values from {rh-openstack} 1.6 and beyond. Consequently, the *Metrics* tab in the *Pipeline* -> *Details* page is removed instead of displaying incorrect values. There is currently no workaround for this issue. (link: https://bugzilla.redhat.com/show_bug.cgi?id=2074767[*BZ#2074767*])
 
+* In {product-title} {product-version}, pod security admission runs globally with privileged enforcement and restricted audit logging and API warnings. Versions of the `opm` binary released earlier than {product-title} {product-version}, do not work with the restricted profile setting. As a result, an error displays if you install a catalog source that was built with an `opm` binary released earlier than {product-title} {product-version}.
++
+.Example error
+[source,terminal]
+----
+Error: open ./db-xxxx: permission denied
+----
++
+Workaround: Cluster administrators can allow the creation of catalog sources built with earlier `opm` versions by setting the pod security admission labels in the namespace metadata files. Apply the `baseline` label to the `warn`, `audit`, and `enforce` modes and set the `scc.podSecurityLabelSync` label to `false`, as shown in the following example:
++
+.Example `manifests/<target_namespace>.yaml` file
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+...
+  labels:
+    security.openshift.io/scc.podSecurityLabelSync: "false"
+    openshift.io/cluster-monitoring: "true"
+    pod-security.kubernetes.io/audit: baseline <1>
+    pod-security.kubernetes.io/warn: baseline <2>
+    pod-security.kubernetes.io/enforce: baseline <3>
```

Comment:
@michaelryanpeter  @anik120 why we are doing these suggestions?
Note that intentionally namespaces prefixed with `openshift-` and a few other ones ( which are created in the installation are considered Openshift system namespace. (see [here](https://github.com/openshift/openshift-docs/pull/48409/files#diff-6b079749a009ff430faa75c3cdece0921ba0e269d1dcd32d6ec0939c7ff99059R11-R23)) Therefore, of these ones is not possible to an enable or disable automatic pod security admission synchronization for most namespaces. 

However, ONLY in 4.12, they will be enforced as restricted. For now, they are privileged. So, I do not understand the relation here with the issue faced. On top of that, I do not think that we should:

a) recommend creating catalog sources or installing Operators on them
b) change the defined enforcement ( they are OCP system namespaces and to aim security we will change the enforcement to restricted on 4.12 )
c) or recommend changing the namespaces directly. We might want to say for users to enable the PSA synchronization. However, I do not think that it has any relation to the issue. 


- We should not recommend users use them to create catalog source
-